internal/ignition/ignition.go

```diff
@@ -492,6 +492,7 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte,
 	envVars := append(os.Environ(),
 		"OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="+g.releaseImage,
 		"OPENSHIFT_INSTALL_INVOKER="+g.installInvoker,
+		"OPENSHIFT_INSTALL_LOAD_CLUSTER_CERTS=true",
```

Comment:
always true?

Reply:
Looks like it's a no-op if the installer fails to find the cert
https://github.com/openshift/installer/blob/cb5373ed8a9960571a2a8acc6bd33fbd6811b577/pkg/asset/tls/certkey.go#L81

Reply:
Indeed, it can't hurt if the files are not present. I updated the code to make it conditional anyway, since there may be some modest security benefit.